project_event/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = ['res.partner']
+
+    client_type = fields.Many2one(
+        'res.partner.category',
+        string='Client Type',
+        related='category_id.client_type',
+        track_visibility='onchange',
+    )
+    sector_id = fields.Many2one(
```

Comment:
@jananjoy we dont need sector_id in the res.partner model

Reply:
@rimbendhaou Remove sector_id in the res.partner model